### PR TITLE
 service card component for home page

### LINKED ISSUE #36: service-card-home-page component
## Common Component - service-card-home-page :
It will be on the homepage and showing services.
- props: data, variant.

- [ ]  It will be used in Pages :
Home page.

- [ ]  Material UI link :
[Card link](https://material-ui.com/components/cards/)

- [ ] UI Image From Design :

image

### PR DESCRIPTION
**Pull request by imansedky5**
fix #36

### CODE REVIEW

File: client/src/component/seviceCardHomePage/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <CardMedia className={classes.media} image={imageSrc} title="service">
+      <Link className={classes.service_name} to="/services">
+        {serviceName}
+      </Link>
+    </CardMedia>
+  );
+};
+
+const { string } = PropTypes;
+ServiceCardForHomePage.propTypes = {
```
Comment (khamis-hesham) [inline client/src/component/seviceCardHomePage/index.js:20]: Good job, I think you should add default values for props

File: client/src/component/seviceCardHomePage/index.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  return (
+    <CardMedia className={classes.media} image={imageSrc} title="service">
+      <Link className={classes.service_name} to="/services">
+        {serviceName}
+      </Link>
+    </CardMedia>
+  );
+};
+
+const { string } = PropTypes;
```
Comment (khamis-hesham) [inline client/src/component/seviceCardHomePage/index.js:19]: Well done, I think it's better to add default values for props

File: client/src/component/seviceCardHomePage/index.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ServiceCardForHomePage.propTypes = {
+  imageSrc: string.isRequired,
+  serviceName: string.isRequired,
+};
```
Comment (khamis-hesham) [inline client/src/component/seviceCardHomePage/index.js:23]: Well done, I think it's better to add default values for props

**Review by mFlaifel — COMMENTED** [review]
- pay attention to font size to make it clear

File: client/src/component/seviceCardHomePage/style.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  media: {
+    height: 200,
+    width: 200,
```
Comment (AdhamHaisami) [inline client/src/component/seviceCardHomePage/style.js:6]: Can you tell me what's unit will be used when not add a specific unit?

**Review by AdhamHaisami — DISMISSED** [review]

File: client/src/component/seviceCardHomePage/style.js (hunk 4 above)
Comment (imansedky5) [reply client/src/component/seviceCardHomePage/style.js:7]: pexels

**Review by mFlaifel — APPROVED** [review]

**Review by yasmeenattallah — APPROVED** [review]

**Review by khamis-hesham — APPROVED** [review]

**Review by AdhamHaisami — APPROVED** [review]

**Pull request merged**